stress_test/class_test/emulator_test_multinet.json

```diff
@@ -16,6 +16,9 @@
 "multinet_topo_group_size":[2],
 "multinet_topo_group_delay_ms":[1000],
 
+"multinet_venv_path":"/opt/venv_multinet",
+"multinet_venv_handler_master":"bin/venv_handler_master.sh",
```

Comment:
does this imply that we need to put venv info within JSON test confs? can't we avoid this?